solidity/contracts/token/TokenBridgeCctpBase.sol

```diff
@@ -71,17 +59,19 @@ contract TokenBridgeCctp is
         address _erc20,
         uint256 _scale,
         address _mailbox,
-        IMessageTransmitter _messageTransmitter,
-        ITokenMessenger _tokenMessenger
+        address _messageTransmitter,
```

Comment:
Looks like the messageTransmitter can be fetched from the tokenMessenger: https://github.com/circlefin/evm-cctp-contracts/blob/5e893866d49326d62d3ecf3f7fb599dd71787469/src/v2/BaseTokenMessenger.sol#L93C29-L93C53